### PR TITLE
Bad parameter 'AU' passed in vector query for CVSS link

### PR DESCRIPTION
**Pull request by csk**
The link was incorrectly passing 'AU' instead of 'Au' parameter to the CVSS link.

Jekyll version I am using (jekyll 3.4.3) generates links as abs urls instead of reltative ones.

### CODE REVIEW

**Comment by csk** [comment]
I just noted that there is some google analytics code being added. The script with the tracking code was already in the codebase but somehow wasn't included until now. I suppose it is my jekyll version. Review.

There is also a .gitignore ignoring _site/* but the files are already being tracked by git so they get comitted too. I think they should be removed from repo but don't know if they are needed by github pages.

File: README.md [hunk 1]
```diff
@@ -19,4 +19,7 @@ For `ssi-injection` it would be:
     url: /templates/ssi-injection
     author: "<your name>"
   ```
+
+1. Install [Jekyll](http://jekyllrb.com/) with `gem install jekyll`
+1. Try out your changes to see if it is working as expected. Serve the project using `jekyll serve --baseurl=/boilerplate-templates`
```
Comment (viyatb) [inline README.md]: You have two "1." indexes there. Please remove the second one.

**Review by viyatb — CHANGES_REQUESTED** [review]
You should have proper, meaningful commit messages.
Try combining the commits (you have 3!) into 1 commit which has a meaningful message.

**Comment by viyatb** [comment]
@csk Actually you can remove the `_site` folder.

1. Run `git rm -rf  --cached _site/`.
2. Commit the delete action to update the PR.

**Comment by viyatb** [comment]
@csk thanks!

**Pull request merged**